pkg/etcdcli/health.go

```diff
@@ -72,12 +69,7 @@ func GetMemberHealth(etcdMembers []*etcdserverpb.Member) memberHealth {
 				}
 				hc.Healthy = true
 			} else {
-				if err == rpctypes.ErrPermissionDenied {
```

Comment:
this was cruft copied from etcdctl `ErrPermissionDenied` is only returned when etcd RBAC is enabled whihc we will never use.